paywall/src/utils/validators.js

```diff
@@ -0,0 +1,14 @@
+// tests whether a field's value was not entered by the user
+export const isNotEmpty = val => val || val === 0
+
+// tests whether a number is positive (0 included) and not a decimal number
```

Comment:
Non-negative to be pedantic 🤔

Otherwise, LG!

Reply:
thanks, that's the verbiage I could not for the life of my find! I'll fix and merge

Reply:
oh, I guess this is in both branches. I'll open an issue for later ;)

Reply:
#1862

Reply:
No worries!